lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1064,17 +1091,20 @@ func (w *WalletKit) prepareSweepParams(in *BumpFeeRequest,
 	// NOTE: if this input doesn't exist and the new budget is not
 	// specified, the params would have a zero budget.
 	params := sweep.Params{
-		Immediate:       immediate,
-		StartingFeeRate: feerate,
-		DeadlineHeight:  deadline,
-		Budget:          budget,
+		Immediate:      immediate,
+		DeadlineHeight: deadline,
+		Budget:         budget,
 	}
 
-	if ok {
-		log.Infof("[BumpFee]: bumping fee for existing input=%v, old "+
-			"params=%v, new params=%v", op, inp.Params, params)
+	// We only set the starting fee rate if it was specified else we keep
+	// the existing one.
+	if !feerate.IsNone() {
+		params.StartingFeeRate = feerate
 	}
 
+	log.Infof("[BumpFee]: bumping fee for existing input=%v, old "+
```

Comment:
Previously this was only logged for an existing sweep. Now we're always logging, but the text mentions "existing input". Seems wrong to me?

Reply:
we would exit early if we do not find the sweep see above case with `!ok`

Reply:
Ah, I missed that. Sgtm then.